File: pythonfiles/City.py
class City:
    def __init__(self, name, latitude, longitude):
        self.name = name
        self.latitude = latitude
        self.longitude = longitude

    def distance_to(self, other_city):
        x_distance = abs(self.latitude - other_city.get_x())
        y_distance = abs(self.longitude - other_city.get_y())
        return ((x_distance ** 2) + (y_distance ** 2)) ** 0.5

    def get_name(self):
        return self.name

    def get_x(self):
        return self.latitude

    def get_y(self):
        return self.longitude

File: src/City.java
package src;
public class City {
    private String name;
    private double latitude;
    private double longitude;

    public City(String name, double latitude, double longitude) {
        this.name = name;
        this.latitude = latitude;
        this.longitude = longitude;
    }
    public double distanceTo(City otherCity) {
        double xDistance = Math.abs(this.latitude - otherCity.getX());
        double yDistance = Math.abs(this.longitude - otherCity.getY());
        return Math.sqrt((xDistance * xDistance) + (yDistance * yDistance));
    }

    public String getName() {
        return name;
    }

    public double getX() {
        return latitude;
    }

    public double getY() {
        return longitude;
    }
}
